covid-19 tracker design changes

alabo-excel opened the pull request:
hello i made changes to the search button, and the background so that the background color changes whenever the page reloads.

alabo-excel committed:
covid-19 tracker design changes

DeltaCoderr commented:
![covid](https://user-images.githubusercontent.com/51528076/94786604-cceb3b00-03ee-11eb-8642-8e5a50c60a3f.PNG)
It shows, Afghanistan for some reason, Can you fix this? the Search button for now looks clean, I like it. The changing colors whenever user refreshes is good too but sometime it changes to the color which matches the boxes whenever we search, which becomes hard to see.

alabo-excel committed:
country name changes

alabo-excel commented:
i have fixed the country name it now shows the correct country and their data.
as for the background that's why i added the light background so the individual boxes can stand out

DeltaCoderr commented:
I would have to close this now as it doesn't resolve conflicts. A new PR was merged with better features, if you come up with fixing that issue, feel free to make a PR, Thanks!
